build.go

```diff
@@ -443,6 +443,9 @@ func (c *Client) processBuildpacks(ctx context.Context, builderBPs []dist.Buildp
 			fetchedBPs = append(fetchedBPs, fetchedBP)
 			order = appendBuildpackToOrder(order, fetchedBP.Descriptor().Info)
 		case buildpack.PackageLocator:
+			// TODO: dynamic assignment of registry to the registry URI
```

Comment:
I believe the daemon can return its default registry url - something like `dockerCli.Info`.

Reply:
I believe this is what we need: https://github.com/moby/moby/blob/fd784ce7c10376247c08dae2083c07ea6300588f/api/types/types.go#L185.